docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc

```diff
@@ -62,17 +62,17 @@ spec:
 '
 ----
 
-. Wait for Compute control plane services' CRs to be ready:
+. Wait for the Compute control plane services CRs to be ready:
```

Comment:
Maybe:
"Wait until the Compute control plane services CRs are ready:"

Ready for what btw?

Reply:
meant to be the CR's 'Ready' status condition is set to True.